tools/buildall

```diff
@@ -78,8 +75,9 @@ def parse_args():
 
 
 def main(args):
-    packagesdir = os.path.abspath(args.dir[0])
-    build_packages(packagesdir, args)
+    packagesdir = Path(args.dir[0]).resolve()
+    outputdir = Path(args.output[0]).resolve()
+    build_packages(packagesdir, outputdir)
```

Comment:
It seems to ignore all the other `args`  from `parse_args` except for `dir` and `output` (e.g.  `cflags`, `ldflags` etc) : is this something we want to do?